packages/nx-plugin/src/executors/playwright/executor.ts

```diff
@@ -0,0 +1,29 @@
+import { execSync } from 'child_process';
+import { ExecutorContext } from '@nx/devkit';
+import { PlaywrightExecutorSchema } from './schema';
+import { playwrightExecutor } from '@nx/playwright';
+
+export default async function runExecutor(
+  options: PlaywrightExecutorSchema,
+  context: ExecutorContext
+) {
+  if (options.testingType !== 'component') {
```

Comment:
based on schema adjustments 

```suggestion
if (options.testingType === 'e2e') {    
```